src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,133 @@ describe("MarketingCollections", () => {
     })
   })
+
+  it("uses the curated sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockResolvedValue({ body: marketingCollectionsData })
+
+    const data = await runQuery(query, context)
+
+    expect(
+      context.marketingCollectionsLoader.mock.calls[0][0]
+    ).not.toContainKeys(["category", "sort"])
+
+    expect(context.marketingCollectionsLoader).toHaveBeenCalledWith({
+      size: 2,
+      slugs: [
+        "new-from-tastemaking-galleries",
+        "new-from-nonprofits-acaf27cc-2d39-4ed3-93dd-d7099e183691",
+        "new-from-small-galleries",
+        "new-from-leading-galleries",
+        "new-to-artsy",
+      ],
+    })
+
+    expect(data).toEqual({
+      marketingCollections: payload,
+    })
+  })
+
+  it("passes the params to the marketing collection loader", async () => {
+    const query = gql`
+      {
+        marketingCollections(
+          size: 2
+          category: "trending"
+          slugs: ["percys-z-collection-1", "fiby-z-collection-2"]
+        ) {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockResolvedValue({ body: marketingCollectionsData })
+
+    const data = await runQuery(query, context)
+
+    expect(context.marketingCollectionsLoader).toHaveBeenCalledWith({
+      size: 2,
+      category: "trending",
+      slugs: ["percys-z-collection-1", "fiby-z-collection-2"],
+    })
+
+    expect(data).toEqual({
+      marketingCollections: payload,
+    })
+  })
+
+  it("throws an error when used with a non-existent category", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 2, category: "newest") {
+          slug
+        }
+      }
+    `
+    const context = {} as any
+    await expect(runQuery(query, context)).rejects.toThrow(
+      "No curated sort available for category: newest"
+    )
+  })
+
+  it("passes when used with a curated sort and without category", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 1) {
+          slug
+        }
+      }
+    `
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockResolvedValue({ body: marketingCollectionsData })
```

Comment:
You can easily collapse these section in a single const.

```suggestion
    const context = {
      marketingCollectionsLoader: jest.fn()
        .mockResolvedValue({ body: marketingCollectionsData }),
    } as any
```

Reply:
The only spec that really needs this mock setup and context defined is the happy path spec. The others introduced here should exercise earlier arg error handling and not rely on context at all.